dbt_subprojects/daily_spellbook/models/_projects/gmx/tokens/arbitrum/gmx_v2_arbitrum_tokens_schema.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+
+models:
+  - name: gmx_v2_arbitrum_CollateralTokensData
```

Comment:
```suggestion
  - name: gmx_v2_arbitrum_collateral_tokens_data
```

Reply:
please also update file name for sql file (aka the model file name)

Reply:
Done